### PR TITLE
Fix autoround format accuracy issue

### PR DESCRIPTION
**Pull request by wenhuach21**

### CODE REVIEW

File: auto_round/export/export_to_autoround/export.py [hunk 1]
```diff
@@ -159,12 +159,12 @@ def save_quantized_as_autoround(output_dir, inplace=True, backend="autoround:exl
     """
     if ":" not in backend:
         backend = "autoround:exllamav2"
+        backend = backend.replace("autoround", "auto_round")
+        backend = backend.replace("auto-round", "auto_round")
```
Comment (WeiweiZhang1) [inline auto_round/export/export_to_autoround/export.py]: Shouldn't these two lines be outside the if ?

File: test/test_export.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
+import copy
+import shutil
+import sys
+import unittest
+
+sys.path.insert(0, "..")
```
Comment (WeiweiZhang1) [inline test/test_export.py:6]: should remove this line

**Review by WeiweiZhang1 — APPROVED** [review]

File: test/test_export.py (hunk 2 above)
Comment (wenhuach21) [reply test/test_export.py:6]: for local test

**Comment by wenhuach21** [comment]
auto-gptq format still has accuracy issue

**Pull request merged**